.gitignore

```diff
@@ -7,20 +7,23 @@ node_modules
 */**/coverage.json
 */**/allFiredEvents
 */**/scTopics
-
+*/**/.coverage_*
```

Comment:
Hmm, is this from the new solidity-coverage version? Can we remove some of the previous lines on ignoring the previous version's coverage output?